website/src/pages/events.tsx

```diff
@@ -0,0 +1,73 @@
+import ArrowRightIcon from 'mdi-react/ArrowRightIcon'
+import * as React from 'react'
+import { Helmet } from 'react-helmet'
+import { ContentSection } from '../components/content/ContentSection'
+import Layout from '../components/Layout'
+import { Tweets } from '../components/Tweets'
+import Conferences from '../components/EventsList'
+export default class EventsPage extends React.Component<any, any> {
+    constructor(props: any) {
+        super(props)
+        this.state = {
+            events: [],
+        }
+    }
+
+    public componentDidMount(): void {
+        if (document) {
+            document.getElementsByTagName('body')[0].setAttribute('style', 'background-image:none')
+        }
+    }
+
+    public render(): JSX.Element | null {
+        const desc =
+            'Meet Sourcegraph at these conferences and meetups around the world.'
+        return (
+            <Layout location={this.props.location}>
+                <div className="Events bg-white text-dark">
+                    <Helmet>
```

Comment:
See https://github.com/sourcegraph/about/blob/master/website/src/pages/case-studies/lyft-monolith-to-microservices.tsx#L8 for an example of not needing Helmet to add page metadata

Reply:
oh yeah, addressed this in latest commit!  thanks.